page-objects/guest-page.js

```diff
@@ -0,0 +1,140 @@
+class Guestacc {
+    async navigateToHomePage() {
+        await page.goto('https://magento2-demo.magebit.com/');
+        await page.waitForTimeout(1000);
+}
+
+    async guestorderpurchase() {
+        await page.click('text=Push It Messenger Bag');
+        await page.waitForTimeout(1000);
+}
+
+async productdetails() {
+    await page.waitForSelector('.catalog-product-view')
+    await page.waitForTimeout(1000);
+}
+
+
+    async addtocart() {
+        await page.click('#product-addtocart-button');
+        await page.waitForTimeout(1000);
+}
+
+    async minicart() {
+        await page.waitForTimeout(3000);
+        await page.click('.minicart-wrapper');
+}
+
+async checkoutbutton() {
+    await page.click('#top-cart-btn-checkout'); 
+    await page.waitForTimeout(1000);
+}
+
+async shippingpage() {
+    await page.waitForSelector('.checkout-shipping-address')
+    await page.waitForTimeout(1000);
+}
+
+
+async nextbuttton() {
+    await page.click('text=Next');
+    await page.waitForTimeout(1000);
+}
+
+async errormessage() {
+    await page.waitForSelector('.table-checkout-shipping-method')
+    await page.waitForTimeout(1000);
+    
+}
+
+async invalidemail() {
+        await page.type('#customer-email', 'karthikeyan39945972gmail.com');
+        await page.waitForTimeout(1000);
+        await page.click('text=Next');
+}
+
+async emailerrormessage() {
+    await page.waitForSelector('#customer-email-error');
+    await page.waitForTimeout(2000);
+}
+
+async invalidZipcode() {
+    await page.type('input[name="postcode"]', 'dsadsfgsd');
+    await page.waitForTimeout(4000);
+
+}
+
+async navigateHomePage() {
+    await page.goto('https://magento2-demo.magebit.com/');
+    await page.waitForTimeout(1000);
+}
+
+async selectproduct() {
+    await page.click('text=Push It Messenger Bag');
+    await page.waitForTimeout(1000);
+}
+
+async pdp() {
+    await page.waitForSelector('.catalog-product-view')
+
+}
+
+async addtocart1() {
+    await page.click('#product-addtocart-button');
+    await page.waitForTimeout(1000);
+}
+
+async minicart1() {
+    await page.waitForTimeout(4000);
+    await page.click('.minicart-wrapper'); 
+}
+
+async checkoutbutton1() {
+    await page.click('#top-cart-btn-checkout'); 
+    await page.waitForTimeout(1000);
+}
+
+async shippingpage1() {
+    await page.waitForSelector('.checkout-shipping-address')
+    await page.waitForTimeout(1000);
+}
+
+
+async validdetails() {
+    await page.type('#customer-email', 'karthikeyan12@gmail.com');
```

Comment:
@karthikeyan7010 You've to give these data in feature file. We can give multiple data providers in it.